core/util/trt_util.cpp

```diff
@@ -238,6 +238,7 @@ const std::unordered_map<at::ScalarType, nvinfer1::DataType>& get_at_trt_type_ma
       {at::kFloat, nvinfer1::DataType::kFLOAT},
       {at::kHalf, nvinfer1::DataType::kHALF},
       {at::kInt, nvinfer1::DataType::kINT32},
+      {at::kLong, nvinfer1::DataType::kINT32},
```

Comment:
I dont want to just associate all instances of long with kINT32. If there are cases of kLong we should explicity handle these cases and explain what we want to do

Reply:
So in this case in the evaluator, we can add a code path for Long in the to evaluator (when truncate is enabled) and print a warning